server/views/pages/licence/HDC_AP.njk

```diff
@@ -80,21 +84,17 @@
    <div id="curfew-address" class="section">
       <p>
          The address(es) to which you are curfewed are:<br>
-         {% if hdcLicenceData.curfewAddress %}
-            <div id="curfewAddressLine1"><span class="bold">{{ hdcLicenceData.curfewAddress.addressLine1 }}</span></div>
-            <div id="curfewAddressLine2"><span class="bold">{{ hdcLicenceData.curfewAddress.addressLine2 }}</span></div>
-            <div id="curfewAddressTown"><span class="bold">{{ hdcLicenceData.curfewAddress.addressTown }}</span></div>
-            <div id="curfewAddressPostCode"><span class="bold">{{ hdcLicenceData.curfewAddress.postCode }}</span></div>
-         {% else %}
-            "MISSING - Curfew Address"
-         {% endif%}
```

Comment:
removed not needed `MISSING` text.

Reply:
not entirely sure whether we need to handle missing addresses, I guess it'll be fine for now